src/twisted/python/usage.py

```diff
@@ -547,9 +547,10 @@ def getUsage(self, width=None):
 
         return s + longdesc + commands
 
-    #def __repr__(self):
```

Comment:
Not a big deal but maybe better just to delete this commented out block

Reply:
Will do.